core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -135,6 +135,9 @@ private TableProperties() {}
   public static final String DELETE_PARQUET_PAGE_ROW_LIMIT = "write.delete.parquet.page-row-limit";
   public static final int PARQUET_PAGE_ROW_LIMIT_DEFAULT = 20_000;
 
+  public static final String PARQUET_DICT_ENABLED = "write.parquet.enable.dictionary";
```

Comment:
This doesn't match the format of Iceberg options. It should be `write.parquet.dict-enabled` to match `dict-size-bytes` or should use `enabled` as the last word if you're using `dictionary` as a part of the hierarchy.

Reply:
+1, the name does match our pattern

Reply:
Apologies for my oversight here, i intended to have `.enabled` in suffix which i mentioned in the table prop but messed up here.